components/sendgrid/sendgrid.app.js

```diff
@@ -1,6 +1,7 @@
 const axios = require("axios");
```

Comment:
Shouldn't we be migrating from `axios` lib to `@pipedream/platform` according to [this doc](https://pipedream.com/docs/pipedream-axios/#pipedream-platform-axios)?

Reply:
Hi @dannyroosevelt @dylburger 
just checking in to see if you want this change actually implemented? when i started coding I remember lib `@pipedream/platform` was meant to be use on the Pipedream UI workflow builder. Now I see it is documented for component actions as well in the documentation, and this is enough reason to doing the change commented by @jcortes 

Now, I understand because of the upcoming release of the new builder, we want to be faster in publishing compoents.
In view of this, do you want this change 

> axios lib to @pipedream/platform
for Sendgrid components?

Reply:
As per this [Slack dicussion](https://pipedream-users.slack.com/archives/G01J50YGJGP/p1645187574309249)
this and other changes noted in [this comment](https://github.com/PipedreamHQ/pipedream/pull/1288#issuecomment-1049088143), I won't be addressing them because they pertain to Sendgrid sources.